lib/Language/Souffle/Interpreted.hs

```diff
@@ -367,7 +367,7 @@ datalogProgramFile prog datalogDir = do
 
 locateSouffle :: IO (Maybe FilePath)
 locateSouffle = do
-  let locateCmd = (shell "which souffle") { std_out = CreatePipe }
+  let locateCmd = (proc "souffle" []) { std_out = CreatePipe }
```

Comment:
I don't think this is correct. Before we searched for souffle, now you are directly invoking it, which is wrong. I may need to add a test specifically for this later..

Can you revert this please?

Reply:
sorry, I had overlooked this. reverting!